src/management/moderation/moderation-manager.service.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+/**
+ * @ngdoc service
+ * @name udb.management.moderation
+ * @description
+ * # Moderation Manager
+ * This service allows you to lookup moderation lists and approve/reject/... Offers.
+ */
+angular
+  .module('udb.management.moderation')
+  .service('ModerationManager', ModerationManager);
+
+/* @ngInject */
+function ModerationManager(udbApi, OfferWorkflowStatus, jobLogger, BaseJob, $q) {
```

Comment:
t was in lijn met de rest van t management stuk (roles, labels, ...), moderation service is idd een betere match denk ik.